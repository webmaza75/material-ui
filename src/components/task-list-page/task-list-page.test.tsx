import * as React from 'react';
import * as ShallowRenderer from 'react-test-renderer/shallow';

import TaskListPage from './task-list-page';

describe(`TaskListPage`, () => {
  it(`TaskListPage renders correctly`, () => {
    const renderer = ShallowRenderer.createRenderer();
    const result = renderer
      .render(<TaskListPage />);

    expect(result).toMatchSnapshot();
  });
});