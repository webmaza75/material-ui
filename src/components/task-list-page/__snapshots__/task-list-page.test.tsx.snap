// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskListPage TaskListPage renders correctly 1`] = `
<React.Fragment>
  <AppPanel
    isOpenConfirmDeleteDialog={false}
    isOpenUndoDeleteSnackbar={false}
    isTaskFormOpen={false}
    onInputChange={[Function]}
    onItemsDelete={[Function]}
    onItemsExactlyDelete={[Function]}
    onItemsUndoDelete={[Function]}
    onSelectionReset={[Function]}
    onTasksCancelDelete={[Function]}
    onTasksConfirmDelete={[Function]}
    onToggleTaskForm={[Function]}
    searchTitle=""
    selected={Array []}
  />
  <div
    style={
      Object {
        "marginBottom": 20,
        "marginLeft": "auto",
        "marginRight": "auto",
        "marginTop": 150,
        "width": "80%",
      }
    }
  >
    <AppTable
      onItemSelect={[Function]}
      selected={Array []}
      taskList={Array []}
    />
  </div>
</React.Fragment>
`;
